library/common/config_template.cc

```diff
@@ -235,6 +236,8 @@ stats_flush_interval: {{ stats_flush_interval_seconds }}s
         - safe_regex:
             google_re2: {}
             regex: '^vhost.api.vcluster\.[\w]+?\.upstream_rq_(?:[12345]xx|retry.*|time|timeout|total)'
+  use_all_default_tags:
+    true
```

Comment:
as envoyproxy/envoy#16125 changes the behavior of metrics_sink, this is necessary to keep the metric names emitted by envoy the way it is, reference: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/stats.proto#envoy-v3-api-msg-config-metrics-v3-statsconfig,

Reply:
You want this to be false to preserve the existing stat names, this defaults to true

Reply:
👍